jlexer/lexer.go

```diff
@@ -253,6 +253,16 @@ func findStringLen(data []byte) (isValid bool, length int) {
 		if idx == 0 || (idx > 0 && data[idx-1] != '\\') {
 			return true, length + idx
 		}
+
+		// count \\\\\\\ sequences. even number of slashes means quote is not really escaped
+		cnt := 1
```

Comment:
maybe set `cnt := 0` and remove `cnt-1` in code below?

Reply:
up to you, i just prefer cnt to mean real "count" rather than smth else.